cdap-ui/app/cdap/components/FastAction/TruncateAction/index.js

```diff
@@ -25,6 +25,7 @@ import ConfirmationModal from 'components/ConfirmationModal';
 import {Tooltip} from 'reactstrap';
 import T from 'i18n-react';
 import classnames from 'classnames';
+import { getType } from '../../../services/metadata-parser/index';
```

Comment:
this should be 
```
import {getType} from 'services/metadata-parser';
```

Reply:
ummm, it seems to work though?

Reply:
fixed

Reply:
yeah using `jquery` also works but its about consistency. We have aliases defined in webpack for use. If we happen to move this file to another location then you don't have to change all the references. also relative paths are hard to follow. Aliases makes them simpler.